packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -53,7 +53,7 @@ describe('src/cypress/dom/visibility', () => {
       expect(fn()).to.be.true
```

Comment:
This file's changes involve undoing https://github.com/cypress-io/cypress/pull/27860